include/pinocchio/spatial/log.hxx

```diff
@@ -87,7 +87,7 @@ namespace pinocchio
 
       {
         Vector3 val_singular;
-        val_singular.array() = 2 * Rnormed.diagonal().array() - tr + Scalar(1);
+        val_singular.array() = Scalar(2) * Rnormed.diagonal().array() - tr + Scalar(1);
```

Comment:
This change is not requested a priori.

Reply:
I can remove this if you ask so, as I double checked this and it's not triggering the issue reported below. However, it looks to me that this might trigger another problem in the future.

Let me know if I should remove it. Note that this answers your request below.

> > 3. Added a missing scalar constructor in `log.hpp`. Without this, Crocoddyl also fails to compile.
> 
> For this issue, could you share the related output of gcc?

Reply:
Remove it